guide/inheritance-wallet/making-changes.md

```diff
@@ -0,0 +1,324 @@
+
+{% include image-gallery.html pages = page.images_key-replacement-add-key %}
+
+#### Cosigner setup, device activation and wallet backup 
+
+The new inheritance wallet is now created. Just like during the initial wallet creation, Alice and Bob now need to:
+
+1. Share the wallet configuration with Alice, so that she can set up the wallet on her app
+1. Activate the new wallet on all of their signing devices
+1. Save the backup kit for the new wallet
+
+We have covered these topics in the previous pages of this reference design.
+
+---
+
+### Moving funds to the new wallet
+
+Now that the new wallet is ready, Alice and Bob need to transfer their savings to this new wallet. Again, our app shows this in the form of a reminder. 
+
+#### Bob creates and signs the transaction
+
+Bob taps on the reminder and initiates the transaction creation process. Because the app is aware of both wallets, it can prefill most of the information in the send flow, such as the recipient address and the transaction amount. After creating the transaction, Bob signs it with his Trezor right away.
+
+{% include image-gallery.html pages = page.images_fund-transfer-bob %} 
+
+The transaction is now ready for Alice to sign as well. However, there is no central server coordinating or automatically syncing transaction information between Bob and Alice's devices, because our app aims to work in the most [self-custodial way]({{ '/guide/inheritance-wallet/overview/#role-of-the-wallet-provider' | relative_url }}) possible. 
+
+This means that Bob has to manually share the [partially signed bitcoin transaction]({{ '/guide/glossary/#partially-signed-bitcoin-transaction-psbt' | relative_url }}) (PSBT) with Alice. Our app offers a variety of different sharing options. Bob chooses to generate a QR code, so that Alice can scan it conveniently.
+
+#### Alice co-signs the transaction
+
+Alice opens the wallet app on her phone, selects the family savings wallet and hits send. In the send sheet she taps "sign transaction" and uses the QR scanner to import the partially signed transaction from Bob's phone.
+
+{% include image-gallery.html pages = page.images_alice-sign-transfer %} 
+
+Once imported, Alice's app displays the transaction details, showing that it has been signed by Bob and requires her signature to complete. Alice reviews the transaction details and initiates the signing process.
+
+After she completes the signing flow  with her BitBox, the app automatically broadcasts the fully signed transaction to the Bitcoin network. If she would like to have more control and broadcast transactions manually, Alice could change this behavior in the app settings.
+
+Back on the home screen, Alice sees that the transaction is being confirmed by the network. After it has been confirmed, the app shows a reminder that the original wallet should be archived. 
+
+---
```

Comment:
I don't totally understand the usage of the divider lines on this page. What's the method to the madness?

Reply:
You're right, I removed them and also tweaked the headline structure in general.